hydroflow_lang/src/graph/ops/switch.rs

```diff
@@ -0,0 +1,50 @@
+use super::{
```

Comment:
Isn't this functionality subsumed by `demux`? If not, should we generalize `demux` to handle enums (including `Either`)?